integration-tests/src/test/resources/sitconfig/java/SitConfigTests.java

```diff
@@ -82,24 +113,30 @@ public static void main(String args[]) throws Exception {
       test.verifyT3ChannelPublicAddress(adminHost);
       test.verifyT3ChannelPublicPort(30091);
     }
+
     if (testName.equals("testCustomSitConfigOverridesForJdbc")) {
-      test.testSystemResourcesJDBCAttributeChange("JdbcTestDataSource-0");
+      String JDBC_URL = args[5];
+      test.testSystemResourcesJDBCAttributeChange("JdbcTestDataSource-0", JDBC_URL);
     }
 
-    if (testName.equals("testSystemResourcesJDBCAttributeChangeSecret")) {
-      test.testSystemResourcesJDBCAttributeChangeSecret("JdbcTestDataSource-1");
-    }
     if (testName.equals("testCustomSitConfigOverridesForJms")) {
       test.testSystemResourcesJMSAttributeChange();
     }
+
     if (testName.equals("testCustomSitConfigOverridesForWldf")) {
       test.testSystemResourcesWLDFAttributeAdd();
     }
   }
 
-  /*    public SitConfigTests(){
-
-  }*/
+  /**
```

Comment:
either write some documentation or remove this - saying exactly what I can read from the code and nothing more is not helpful

Reply:
Added some more doc.